.github/workflows/bats.yaml

```diff
@@ -27,6 +27,8 @@ on:
         description: Package run ID override; leave empty to use latest.
         default: ''
         type: string
+  schedule:
+  - cron: '0 8 * * 1-5' # 8AM UTC weekdays as a baseline
```

Comment:
I would have put this before `workflow_dispatch`, just to make it easier to verify that the indentation is correct. 😄